gdal/alg/gdalwarpkernel.cpp

```diff
@@ -6110,7 +6178,8 @@ static void GWKAverageOrModeThread( void* pData)
                 else if( nAlgo == GWKAOM_Max )
                 // poWK->eResample == GRA_Max.
                 {
-                    dfTotal = std::numeric_limits<double>::lowest();
+                    dfTotalReal = std::numeric_limits<double>::lowest();
```

Comment:
I'm unsure about the wanted semantics of max and min for complex values. Mathematically speaking there is no obvious way to compare 2 complex values

Reply:
Agreed. This is unclear. Should I resort to min/max of abs values or individual min/max for the real and imaginary parts. Either are acceptable solutions as long as we document it. The same applies for quantiles too.

Reply:
Does it really makes sense to support those modes for complex values if there isn't an obvious definition for them ? Do you have use cases ?

Reply:
I'm also happy to throw an error or warning for min/max/quantiles. There are no use cases for min/ max / quantiles but I definitely have use cases for average (multi-looking of complex SAR/InSAR datasets).

Reply:
Erroring out for min/max/quantiles seem reasonable to me. I agree that averaging makes sense.

Reply:
Great. Will work on the changes and get back. Might take me a week or so.